index.html

```diff
@@ -0,0 +1,130 @@
+                </li>
+                <li>
+                    <p><strong>1938</strong> - Marries wife of 69 years Margret Gibson. Gets laid off due to budget
+                        cuts. Inspired
+                        by Elvin Charles Stakman, he returns to school study under Stakman, who teaches him about
+                        breeding
+                        pest-resistent plants.</p>
+                </li>
+                <li>
+                    <p><strong>1941</strong> - Tries to enroll in the military after the Pearl Harbor attack, but is
+                        rejected.
+                        Instead, the military asked his lab to work on waterproof glue, DDT to control malaria,
+                        disinfectants, and
+                        other applied science.</p>
+                </li>
+                <li>
+                    <p><strong>1942</strong> - Receives a Ph.D. in Genetics and Plant Pathology</p>
+                </li>
+                <li>
+                    <p><strong>1944</strong> - Rejects a 100% salary increase from Dupont, leaves behind his pregnant
+                        wife, and
+                        flies to Mexico to head a new plant pathology program. Over the next 16 years, his team breeds
+                        6,000
+                        different strains of disease resistent wheat - including different varieties for each major
+                        climate on
+                        Earth.</p>
+                </li>
+                <li>
+                    <p><strong>1945</strong> - Discovers a way to grown wheat twice each season, doubling wheat yields
+                    </p>
+                </li>
+                <li>
+                    <p>1953- crosses a short, sturdy dwarf breed of wheat with a high-yeidling
+                        American breed,
+                        creating a strain that responds well to fertilizer. It goes on to provide 95% of Mexico's wheat.
+                    </p>
+                    <p><strong>1962</strong> - Visits Delhi and brings his high-yielding strains of wheat to the Indian
+                        subcontinent in time to help mitigate mass starvation due to a rapidly expanding population</p>
+                    <!-- <p><strong>1968</strong> - Visits Delhi and brings his high-yielding strains of wheat to the Indian
+                        subcontinent in time to help mitigate mass starvation due to a rapidly expanding population</p> -->
+                </li>
+                <li>
+                    <p><strong>1970</strong> - receives the Nobel Peace Prize</p>
+                </li>
+                <li>
+                    <p><strong>1983</strong> - helps seven African countries dramatically increase their maize and
+                        sorghum yields
+                    </p>
+                </li>
+
+                <p>1984 - becomes a distinguished professor at Texas A&M University
+                </p>
+
```

Comment:
- [ ] This `p` tag is inconsistent with other `li`s. wrap this `p` tag with the `li` tag.

Reply:
Nice job pointing this out :+1:. However, as I have stated in a previous comment, I suggest you phrase your comments politely as suggestions instead of orders to sound more friendly as well as elaborate more on why you requested this change. You can refer to this [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate) on how you can communicate better when doing reviews.